session-02-big-picture/README.md

```diff
@@ -257,6 +257,44 @@ Possibly prerecorded videos. Need offline support
        ```git clone https://github.com/koushikkothagal/java-backend-bootcamp.git```
     
 
+## On Linux
+
+Linux comes with git pre-installed, you can check the version of git by typing the following command in terminal
+
+```console
+git --version
+```
+![git version](./assets/git-linux-setup/git-version.png)
```

Comment:
I suggest removing the screenshot. Just add the output in a code fence block. Like so
~~~
```console
$ git --version
git version 2.37.1
```
~~~

Notice the `$ ` to indicate prompt.